Automattic/crowdsignal-ui
# Add 'finish resolution' action

This PR adds FINISH_RESOLUTION action type/creator to actively, well, finish the resolve resolution and avoid a full re-render of the editor.

NOTE: I found no way to import the action creator from the `@wordpress/data` package (no specific export, full path: https://github.com/WordPress/gutenberg/blob/trunk/packages/data/src/redux-store/metadata/actions.js), if there's a way, I'd rather have it imported instead of duped.

## Test instructions
Create a new project. See that upon first save the editor doesn't do a full reload (it should be evident if it does as the loader takes around 2 seconds when this happens).
Publish the project. The PublishButton, Share and Preview should respond as expected
